### PR TITLE
Removed nested-sitemapindex

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | https://github.com/sulu/SuluArticleBundle/pull/34
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR merges the provider sitemapindex and the root-sitemapindex.

#### Why?

Google webmaster-tools throw warnings because of nested sitemap-indexes.

### CODE REVIEW

**Comment by danrot** [comment]
* We should only create the sitemap for portal informations being of type `FULL_MATCH`, otherwise also custom urls will be loaded, which have no locale, which causes an error.
* The check for the host in `dumpSitemap` for `strpos` needs to be false, not `-1`.

**Comment by wachterjohannes** [comment]
@danrot fixed

File: src/Sulu/Bundle/WebsiteBundle/Command/DumpSitemapCommand.php [hunk 1]
```diff
@@ -93,7 +94,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $output->writeln('Start dumping "sitemap.xml" files:');
 
         $webspaceManager = $this->getContainer()->get('sulu_core.webspace.webspace_manager');
+
+        /** @var PortalInformation $portalInformation */
```
Comment (danrot) [inline src/Sulu/Bundle/WebsiteBundle/Command/DumpSitemapCommand.php]: From what I remember we always  put these comments in the foreach block, don't we?

**Comment by danrot** [comment]
It looks to me like the sitemap is never read from the cache... There is no `sitemap.xml` file in the folder the `SitemapController` is doing the lookup. It's looking for it in `sulu-standard/app/cache/website/dev/sulu/sitemaps/http/sulu.lo`, but there is another folder separating the different languages.

**Pull request merged**